scripts/aws-deploy.sh

```diff
@@ -0,0 +1,288 @@
+
+echo $SMTP_HOST | grep -Eo "([^.*.*]+)"
+
+# -------------------------------- Simcore -------------------------------
+
+pushd ${repo_basedir}/services/simcore;
+
+ori_env_simcore=`cat .env`
+
+# Set the image tag to be used from dockerhub
+$psed -i -e "s/DOCKER_IMAGE_TAG=.*/DOCKER_IMAGE_TAG=$SIMCORE_IMAGE_TAG/" .env
+
+# Hostnames
+$psed -i -e "s/MONITORING_DOMAIN=.*/MONITORING_DOMAIN=$MONITORING_DOMAIN/" .env
+$psed -i -e "s/PUBLISHED_HOST_NAME=.*/PUBLISHED_HOST_NAME=$MACHINE_FQDN/" .env
+
+# PGSQL
+$psed -i -e "s/POSTGRES_DB=.*/POSTGRES_DB=$POSTGRES_DB/" .env
+$psed -i -e "s/POSTGRES_ENDPOINT=.*/POSTGRES_ENDPOINT=$POSTGRES_ENDPOINT/" .env
+$psed -i -e "s/POSTGRES_HOST=.*/POSTGRES_HOST=$POSTGRES_HOST/" .env
+$psed -i -e "s/POSTGRES_USER=.*/POSTGRES_USER=$POSTGRES_USER/" .env
+$psed -i -e "s/POSTGRES_PASSWORD=.*/POSTGRES_PASSWORD=$POSTGRES_PASSWORD/" .env
+$psed -i -e "s/POSTGRES_PORT=.*/POSTGRES_PORT=$POSTGRES_PORT/" .env
+
+# Registry
+$psed -i -e "s/REGISTRY_AUTH=.*/REGISTRY_AUTH=$REGISTRY_AUTH/" .env
+$psed -i -e "s/REGISTRY_PW=.*/REGISTRY_PW=$REGISTRY_PW/" .env
+$psed -i -e "s/REGISTRY_SSL=.*/REGISTRY_SSL=$REGISTRY_SSL/" .env
+$psed -i -e "s/REGISTRY_URL=.*/REGISTRY_URL=$REGISTRY_DOMAIN/" .env
+$psed -i -e "s/REGISTRY_USER=.*/REGISTRY_USER=$REGISTRY_USER/" .env
+
+# S3
+$psed -i -e "s/S3_ACCESS_KEY=.*/S3_ACCESS_KEY=$ACCESS_KEY_ID/" .env
+$psed -i -e "s~S3_SECRET_KEY=.*~S3_SECRET_KEY=$SECRET_ACCESS_KEY~" .env
+$psed -i -e "s/S3_BUCKET_NAME=.*/S3_BUCKET_NAME=$S3_BUCKET/" .env
+$psed -i -e "s/S3_ENDPOINT=.*/S3_ENDPOINT=$S3_ENDPOINT/" .env
+$psed -i -e "s/S3_SECURE=.*/S3_SECURE=$S3_SECURE/" .env
+
+# mail
+$psed -i -e "s/SMTP_HOST=.*/SMTP_HOST=$SMTP_HOST/" .env
+$psed -i -e "s/SMTP_PORT=.*/SMTP_PORT=$SMTP_PORT/" .env
+$psed -i -e "s/SMTP_USERNAME=.*/SMTP_USERNAME=$SMTP_USERNAME/" .env
+$psed -i -e "s/SMTP_PASSWORD=.*/SMTP_PASSWORD=$SMTP_PASSWORD/" .env
+
+# Osparc config
+$psed -i -e "s/WEBSERVER_LOGIN_REGISTRATION_CONFIRMATION_REQUIRED=.*/WEBSERVER_LOGIN_REGISTRATION_CONFIRMATION_REQUIRED=$WEBSERVER_LOGIN_REGISTRATION_CONFIRMATION_REQUIRED/" .env
+$psed -i -e "s/WEBSERVER_LOGIN_REGISTRATION_INVITATION_REQUIRED=.*/WEBSERVER_LOGIN_REGISTRATION_INVITATION_REQUIRED=$WEBSERVER_LOGIN_REGISTRATION_INVITATION_REQUIRED/" .env
+$psed -i -e "s/WEBSERVER_STUDIES_ACCESS_ENABLED=.*/WEBSERVER_STUDIES_ACCESS_ENABLED=$WEBSERVER_STUDIES_ACCESS_ENABLED/" .env
+
+# Rabbit
+$psed -i -e "s/RABBIT_PORT=.*/RABBIT_PORT=$RABBIT_PORT/" .env
+$psed -i -e "s/RABBIT_HOST=.*/RABBIT_HOST=$RABBIT_HOST/" .env
+$psed -i -e "s/RABBIT_LOG_CHANNEL=.*/RABBIT_LOG_CHANNEL=$RABBIT_LOG_CHANNEL/" .env
+$psed -i -e "s/RABBIT_PROGRESS_CHANNEL=.*/RABBIT_PROGRESS_CHANNEL=$RABBIT_PROGRESS_CHANNEL/" .env
+$psed -i -e "s/RABBIT_USER=.*/RABBIT_USER=$RABBIT_USER/" .env
+$psed -i -e "s/RABBIT_PASSWORD=.*/RRABBIT_PASSWORD=$RABBIT_PASSWORD/" .env
+
+# Reddis
+$psed -i -e "s/REDIS_HOST=.*/REDIS_HOST=$REDIS_HOST/" .env
+$psed -i -e "s/REDIS_PORT=.*/REDIS_PORT=$REDIS_PORT/" .env
```

Comment:
would it not be simpler to either:
1. export all the entries in repo.config? this would override the ones in .env no?

2. echo repo.config >> .env that would add all the values at the end of .env and overwrite all of them

I am pretty sure at least 2. would work. then you do not need to change that script everytime we add a variable